File: translation_output/starcoder/codenet/java/python-sanitized/p02331.py
import sys

MOD=1000000007

def main():
	sc=sys.stdin
	#åºå¥ã§ãããã¼ã«ãåºå¥ã§ããç®±ã«å
¥ããå
¥ãæ¹ã¯ã
	#1ã¤ã®ãã¼ã«ã«ä»ããç®±ã®åæ°éãæããããããç¬ç«ãªã®ã§ã
	#n kãä¸ããããæ k^nã§ããã
	n=int(sc.readline())
	k=int(sc.readline())
	ans=1
	for i in range(n):
		ans=(ans*k)%MOD
	print(ans)

if __name__ == '__main__':
	main()

File: translation_output/no_test/wizardcoder/codenet/python/java/p02331.java
Below is an instruction that describes a task. Write a response that appropriately completes the request.

### Instruction:
Translate the following python code to java and enclose your solution inside ```java```:
```
#!/usr/bin/env python

import sys
import math
import itertools as it
from collections import deque

sys.setrecursionlimit(10000000)

MOD = 10 ** 9 + 7

n, k = map(int, raw_input().split())

print k ** n % MOD

```

### Response:
```java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int k = sc.nextInt();
        int MOD = 1000000007;
        int ans = 1;
        for (int i = 1; i <= n; i++) {
            ans = (ans * k) % MOD;
        }
        System.out.println(ans);
    }
}
```<|endoftext|>
